manifests/init.pp

```diff
@@ -309,6 +310,9 @@
   # Setup any DHCP pools for IPv4
   create_resources('dhcp::pool', $pools)
 
+  # Setup any DHCP shared-networks
+  create_resources('dhcp::sharednetwork', $sharednetworks)
```

Comment:
Calling `create_resources` is deprecated. Please iterate over the Hash and create the resources instead.

```suggestion
  # Setup any DHCP shared-networks
  $sharednetworks.each |$sharednetwork, $sharednetworkattr|
  dhcp::sharednetwork { $sharednetwork:
    * => $sharednetworkattr,
  }
```

Reply:
Can you give me a pointer on why create_resources() got deprecated (asking of curiosity since we use that ourselves often). I know that `ensure_resource()` shouldn't be used.

Nevertheless, I will rework this bit. Guess I leave the other create_resources() unattached, since that would be out of scope for this PR.